.github/workflows/ci.yml

```diff
@@ -58,15 +58,39 @@ jobs:
   test:
     name: test
     runs-on: ubuntu-latest
+    env:
```

Comment:
I think I'd be more inclined to just cache things like we do in other projects (e.g. the compiler), rather than bring `sccache` into the mix, since I don't think we're likely to see significant benefits doing so versus the simpler approach.

Either way, it's key that we're able to bust the cache manually, as we inevitably will need to do so at some point.